### ISSUE TITLE
Problems with iOS 13.3?

### ISSUE THREAD

**Issue by michaeldupuis** [huynguyencong/ImageScrollView#54]
Is anyone else seeing issues since upgrading to iOS 13? The image often doesn't seem to be centered now, especially when running in the iPhone 11 simulator. Note the big gap at the top, which wasn't there before. It should look like the second image by default.

<img width="517" alt="Screen Shot 2019-12-18 at 6 27 12 PM" src="https://user-images.githubusercontent.com/22894020/71131994-4f8a6d00-21c4-11ea-85d8-5c635ca569af.png">
<img width="517" alt="Screen Shot 2019-12-18 at 6 27 29 PM" src="https://user-images.githubusercontent.com/22894020/71131995-4f8a6d00-21c4-11ea-99af-7f74efc8b66a.png">

**Comment by huynguyencong** [comment]
Do you still have the problem?

**Comment by bilaalrashid** [comment]
Yes

**Comment by bilaalrashid** [comment]
I have tried to investigate this issue before but I wasn't able to find exactly what causes the issue to be able to reproduce it so I haven't been able to put it in a sharable example, although I may try again if I have time at some point in the next few days. Like #57 (maybe also worth merging the two issues to keep this all in the same place), I have found that it may be related to when it's embedded in a `UINavigationController`. However, it affects me with any size of image - both portrait or horizontal. 

One weird thing to add to this is that after using around with this in my app a lot, very, very occasionally (maybe only 2 or 3 times out of hundreds or thousands), it has worked as it should. My app uses taps to hide/show the navigation bar so may be related to this as that has happened after hiding and showing the navigation bar, zooming in and out and panning around the image a lot.

This is still happening to me on iOS 13.6.0. I'm not running the iOS 14 or Xcode 12 betas yet, but I was wondering if anyone else who is, is still experiencing these issues on iOS 14? It would make a big difference into how much of a problem this is and could maybe help a bit in finding the cause.
